<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmDialogWidget</class>
 <widget class="QWidget" name="ConfirmDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="4" column="0">
    <widget class="QGroupBox" name="other_groupbox">
     <property name="title">
      <string>Other</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="force_dark_cbx">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Force dark current</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="skip_existing_images_cbx">
        <property name="text">
         <string>Skip already collected images</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="take_video_cbx">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Rotate sample by 360 degrees and record video</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="take_snapshots_hlayout">
        <item>
         <widget class="QLabel" name="take_snapshots_label">
          <property name="text">
           <string>Number of crystal snapshots:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="take_snapshots_combo">
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="buttons_layout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="continue_button">
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="file_gbox">
     <property name="title">
      <string>Existing files that will be overwritten (first 20 images are checked):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="file_treewidget">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Sample</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Directory</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File name</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="summary_gbox">
     <property name="title">
      <string>Summary</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="1" column="0">
       <widget class="QTreeWidget" name="summary_treewidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <property name="headerHidden">
         <bool>false</bool>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Method</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Sample</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Directory</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Template</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Energy</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Resolution</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Transmission</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Osc. start</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Osc. range</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Images</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Exp. time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Total osc.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Total exp. time</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="summary_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="2" column="0">
      <widget class="QCheckBox" name="inverse_cbx">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Inverse beam subwedge size:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="inverse_beam_num_images_ledit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="interleave_cbx">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Interleave subwedge size:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="interleave_images_num_ledit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>interleave_cbx</sender>
   <signal>toggled(bool)</signal>
   <receiver>interleave_images_num_ledit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inverse_cbx</sender>
   <signal>toggled(bool)</signal>
   <receiver>inverse_beam_num_images_ledit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>581</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
